### PR TITLE
Updating .NET SDK snippets

### PR DESCRIPTION
**Pull request by delias-silva**
Update of the .NET SDK snippets to be compatible with the new version 2.0.0 of the SDK.

### Checklist:
Before sending your contribution, please specify the following: 

- [x] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/online-payments/checkout-pro/advanced-integration.en.md [hunk 1]
```diff
@@ -114,33 +114,33 @@ payer = MercadoPago::Payer.new({
...
     {
         AreaCode = "",
         Number = "949 128 866"
     },
     ----[mla, mlb, mlu, mco, mlc, mpe]----
-    Identification = new Identification()
+    Identification = new IdentificationRequest
     {
         Type = "DNI",
         Number = "12345678"
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/advanced-integration.en.md:135]:
```suggestion
        Number = "12345678",
```

File: guides/online-payments/checkout-pro/advanced-integration.en.md [hunk 2]
```diff
@@ -114,33 +114,33 @@ payer = MercadoPago::Payer.new({
...
+var payer = new PreferencePayerRequest
 {
     Name = "Charles",
     Surname = "Luevano",
     Email = "charles@hotmail.com",
-    Phone = new Phone()
+    Phone = new PhoneRequest
     {
         AreaCode = "",
         Number = "949 128 866"
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/advanced-integration.en.md:129]:
```suggestion
        Number = "949 128 866",
```

File: guides/online-payments/checkout-pro/advanced-integration.es.md [hunk 3]
```diff
@@ -114,33 +114,33 @@ payer = MercadoPago::Payer.new({
...
+var payer = new PreferencePayerRequest
 {
     Name = "Charles",
     Surname = "Luevano",
     Email = "charles@hotmail.com",
-    Phone = new Phone()
+    Phone = new PhoneRequest
     {
         AreaCode = "",
         Number = "949 128 866"
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/advanced-integration.es.md:129]:
```suggestion
        Number = "949 128 866",
```

File: guides/online-payments/checkout-pro/advanced-integration.es.md [hunk 4]
```diff
@@ -114,33 +114,33 @@ payer = MercadoPago::Payer.new({
...
     },
     ----[mla, mlb, mlu, mco, mlc, mpe]----
-    Identification = new Identification()
+    Identification = new IdentificationRequest
     {
         Type = "DNI",
         Number = "12345678"
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/advanced-integration.es.md:135]:
```suggestion
        Number = "12345678",
```

File: guides/online-payments/checkout-pro/advanced-integration.pt.md [hunk 5]
```diff
@@ -114,33 +114,33 @@ payer = MercadoPago::Payer.new({
...
-    Phone = new Phone()
+    Name = "Charles",
+    Surname = "Luevano",
+    Email = "charles@hotmail.com",
+    Phone = new PhoneRequest
     {
-        AreaCode = "11",
-        Number = "4444-4444"
+        AreaCode = "",
+        Number = "949 128 866"
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/advanced-integration.pt.md:129]:
```suggestion
        Number = "949 128 866",
```

File: guides/online-payments/checkout-pro/advanced-integration.pt.md [hunk 6]
```diff
@@ -114,33 +114,33 @@ payer = MercadoPago::Payer.new({
...
     },
     ----[mla, mlb, mlu, mco, mlc, mpe]----
-    Identification = new Identification()
+    Identification = new IdentificationRequest
     {
-        Type = "CPF",
-        Number = "19119119100"
+        Type = "DNI",
+        Number = "12345678"
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/advanced-integration.pt.md:135]:
```suggestion
        Number = "12345678",
```

File: guides/online-payments/checkout-pro/shipments.en.md [hunk 7]
```diff
@@ -616,41 +655,50 @@ preference.save
...
-
-preference.Payer = payer;
-preference.Shipments = shipments;
-
-preference.Save();
+        new PreferenceItemRequest
+        {
+            Title = "Blue shirt",
+            Quantity = 1,
+            UnitPrice = [FAKER][COMMERCE][PRICE]m,
```
Comment (nicolassalvanes) [inline guides/online-payments/checkout-pro/shipments.en.md:666]: Excelente esto! Buen ojo! 😄

File: guides/online-payments/web-tokenize-checkout/customers-and-cards.en.md [hunk 8]
```diff
@@ -214,8 +213,8 @@ You can get the complete list of `Cards` from a customer by doing a `HTTP GET` r
 
 ```
 ```csharp
-customer = Customer.FindById("customer.Id");
-List<Card> cards = customer.Cards; 
+var customerClient = new CustomerClient();
+ResourcesList<CustomerCard> results = await customerClient.ListCardsAsync("CUSTOMER_ID");
```
Comment (nicolassalvanes) [inline guides/online-payments/web-tokenize-checkout/customers-and-cards.en.md:217]:
```suggestion
ResourcesList<CustomerCard> customerCards = await customerClient.ListCardsAsync("CUSTOMER_ID");
```

File: guides/online-payments/web-tokenize-checkout/customers-and-cards.en.md [hunk 9]
```diff
@@ -535,8 +538,8 @@ You can get the complete list of `Cards` of a client by making a `HTTP GET` requ
 
 ```
 ```csharp
-Customer customer = Customer.FindById("customer.Id");
-List<Card> cards = customer.Cards;
+var customerClient = new CustomerClient();
+ResourcesList<CustomerCard> results = await customerClient.ListCardsAsync("CUSTOMER_ID");
```
Comment (nicolassalvanes) [inline guides/online-payments/web-tokenize-checkout/customers-and-cards.en.md:542]:
```suggestion
ResourcesList<CustomerCard> customerCards = await customerClient.ListCardsAsync("CUSTOMER_ID");
```

File: guides/online-payments/web-tokenize-checkout/customers-and-cards.es.md [hunk 10]
```diff
@@ -214,8 +213,8 @@ Puedes obtener el listado completo de `Cards` de un cliente realizando un reques
 
 ```
 ```csharp
-customer = Customer.FindById("customer.Id");
-List<Card> cards = customer.Cards; 
+var customerClient = new CustomerClient();
+ResourcesList<CustomerCard> results = await customerClient.ListCardsAsync("CUSTOMER_ID");
```
Comment (nicolassalvanes) [inline guides/online-payments/web-tokenize-checkout/customers-and-cards.es.md:217]:
```suggestion
ResourcesList<CustomerCard> customerCards = await customerClient.ListCardsAsync("CUSTOMER_ID");
```

File: guides/online-payments/web-tokenize-checkout/customers-and-cards.es.md [hunk 11]
```diff
@@ -535,8 +538,8 @@ Puedes obtener el listado completo de `Cards` de un cliente realizando un reques
 
 ```
 ```csharp
-Customer customer = Customer.FindById("customer.Id");
-List<Card> cards = customer.Cards;
+var customerClient = new CustomerClient();
+ResourcesList<CustomerCard> results = await customerClient.ListCardsAsync("CUSTOMER_ID");
```
Comment (nicolassalvanes) [inline guides/online-payments/web-tokenize-checkout/customers-and-cards.es.md:542]:
```suggestion
ResourcesList<CustomerCard> customerCards = await customerClient.ListCardsAsync("CUSTOMER_ID");
```

File: guides/online-payments/web-tokenize-checkout/customers-and-cards.pt.md [hunk 12]
```diff
@@ -212,8 +211,8 @@ Podendo obter uma lista completa de `Cards` de um cliente realizando um request
 
 ```
 ```csharp
-customer = Customer.FindById("customer.Id");
-List<Card> cards = customer.Cards; 
+var customerClient = new CustomerClient();
+ResourcesList<CustomerCard> results = await customerClient.ListCardsAsync("CUSTOMER_ID");
```
Comment (nicolassalvanes) [inline guides/online-payments/web-tokenize-checkout/customers-and-cards.pt.md:215]:
```suggestion
ResourcesList<CustomerCard> customerCards = await customerClient.ListCardsAsync("CUSTOMER_ID");
```

File: guides/online-payments/web-tokenize-checkout/customers-and-cards.pt.md [hunk 13]
```diff
@@ -533,8 +536,8 @@ Poderá obter uma lista completa de `Cards` de um cliente realizando um request
 
 ```
 ```csharp
-Customer customer = Customer.FindById("customer.Id");
-List<Card> cards = customer.Cards;
+var customerClient = new CustomerClient();
+ResourcesList<CustomerCard> results = await customerClient.ListCardsAsync("CUSTOMER_ID");
```
Comment (nicolassalvanes) [inline guides/online-payments/web-tokenize-checkout/customers-and-cards.pt.md:540]:
```suggestion
ResourcesList<CustomerCard> customerCards = await customerClient.ListCardsAsync("CUSTOMER_ID");
```

**Review by nicolassalvanes — CHANGES_REQUESTED** [review]
Buen trabajo @delias-silva ! 💪  Dejé algunos comentarios y sugerencias, si algo no se entiende por favor me avisas.

File: guides/online-payments/checkout-api/other-features.en.md [hunk 14]
```diff
@@ -100,6 +100,30 @@ payment.payer = {
...
+    PaymentMethodId = "visa",
+    Payer = new PaymentPayerRequest
+    {
+        Email = "test_user_19653727@testuser.com",
+    },
+    Capture = false,
+};
+
+var client = new PaymentClient();
+Payment payment = await client.CreateAsync(paymentRequest);
```
[the remark this answers is not in the document]
Comment (delias-silva) [reply guides/online-payments/checkout-api/other-features.en.md:125]: Sí, creo que es el mejor. Voy a cambiar donde no está así.

File: guides/online-payments/checkout-api/other-features.en.md [hunk 15]
```diff
@@ -192,6 +217,15 @@ payment = MercadoPago::Payment.load(paymentId)
...
 payment.update()
 ```
+```csharp
+using MercadoPago.Client.Payment;
+using MercadoPago.Config;
+
+MercadoPagoConfig.AccessToken = "ENV_ACCESS_TOKEN";
+
+var client = new PaymentClient();
+var payment = await client.CaptureAsync(paymentId);
```
[the remark this answers is not in the document]
Comment (delias-silva) [reply guides/online-payments/checkout-api/other-features.en.md:227]: Sí, voy a cambiar.

**Review by nicolassalvanes — APPROVED** [review]
Excelente! Veo todo bien 👌

**Review by nicolassalvanes — APPROVED** [review]

**Pull request merged**